### PR TITLE
[No JIRA] Add pull request template

### PR DESCRIPTION
**Pull request by k0nserv**
cc @tvooo, @matthewdavidson

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Thanks for your PR. Please verify below that you've made all the necessary changes.
+
++ [ ] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
++ [ ] For any new files I've included the Apache 2.0 License header
```
Comment (tvooo) [inline .github/PULL_REQUEST_TEMPLATE.md]: Link to the header / file would be great here

Comment (k0nserv) [inline .github/PULL_REQUEST_TEMPLATE.md]: Good idea :)

**Comment by k0nserv** [comment]
I'm open to more thing we should add in this template

**Comment by tvooo** [comment]
We can add things as we discover them missing in new PRs :) Happy with this for now.

**Pull request merged**